src/Mailgun/Api/Message.php

```diff
@@ -30,18 +30,21 @@ public function send($domain, array $params)
         Assert::notEmpty($domain);
         Assert::notEmpty($params);
 
+        $meme = '';
+        if (!empty($params['message'])) {
+            $meme = '.mime';
+        }
+
         $postDataMultipart = [];
         $fields = ['message', 'attachment', 'inline'];
         foreach ($fields as $fieldName) {
             if (!isset($params[$fieldName])) {
                 continue;
             }
-            if (!is_array($params[$fieldName])) {
```

Comment:
This can be removed since if this is not an array, we would get a type error calling prepareFile.